src/VisualStudio/CSharp/Impl/Options/Formatting/StyleViewModel.cs

```diff
@@ -1777,6 +1781,54 @@ class Class
     //]
     }}
 }}
+";
+
+        private static readonly string s_allow_bank_line_after_token_in_conditional_expression_true = $@"
+class Class
+{{
+//[
+    // {ServicesVSResources.Allow_colon}
+    void Method()
+    {{
+        var result = CheckCondition() ?
+            WhenTrue :
+            WhenFalse;
+    }}
+//]
+}}
+";
+
+        private static readonly string s_allow_bank_line_after_token_in_conditional_expression_false = $@"
+namespace NS1
+{{
+    class Class
+    {{
+    //[
+        // {ServicesVSResources.Require_colon}
+        void Method()
+        {{
+            var result = CheckCondition()
+                ? WhenTrue
+                : WhenFalse;
+        }}
+    //]
+    }}
+}}
+namespace NS2
+{{
+    class Class
+    {{
+    //[
+        // {ServicesVSResources.Over_colon}
```

Comment:
Omit?

```suggestion
        // {ServicesVSResources.Omit_colon}
```

Reply:
the "require x over y" pattern is what we use for all the other options.